fog-threat-model-2.0.0.md

```diff
@@ -128,7 +128,6 @@ Eve:
   (This is the current threat model for MoblieCoin consensus server.)
 
 * Otherwise, Eve obeys the SAME set of invariants as described for Oscar, with the following modifications:
-  * Eve can determine the number of new payment records retrieved during balance checking, but not precisely which payment records belong to the user.
```

Comment:
thank you! can we also increase the version number of this file, using like `git mv`?
I think it should now be `2.1`